src/Symfony/Component/HttpFoundation/Tests/IpUtilsTest.php

```diff
@@ -30,13 +30,12 @@ public function testIpv4Provider()
             array(true, '192.168.1.1', '192.168.1.1/1'),
             array(true, '192.168.1.1', '192.168.1.0/24'),
             array(false, '192.168.1.1', '1.2.3.4/1'),
-            array(false, '192.168.1.1', '192.168.1/33'),
+            array(false, '192.168.1.1', '192.168.1.1/33'), // invalid subnet
             array(true, '192.168.1.1', array('1.2.3.4/1', '192.168.1.0/24')),
             array(true, '192.168.1.1', array('192.168.1.0/24', '1.2.3.4/1')),
             array(false, '192.168.1.1', array('1.2.3.4/1', '4.3.2.1/1')),
-            array(true, '1.2.3.4', '0.0.0.0/0'),
```

Comment:
The case verifying `1.2.3.4` is in subnet `192.168.1.0/0` covers this code path, but I'll add it back anyway.